env0/resource_project_test.go

```diff
@@ -0,0 +1,66 @@
+package env0
+
+import (
+	"fmt"
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+func TestUnitProjectResource(t *testing.T) {
```

Comment:
```suggestion
func TestUnitProjectResource(test *testing.T) {
```

Reply:
That's go specification, the signature is always `t *testingT`